### ISSUE TITLE
learning rate curve

### ISSUE THREAD

**Issue by together0** [pengzhiliang/MAE-pytorch#87]
Why learning rate from 0 to 1e-4?  Shouldn't it be 1e-4 to 0?
